src/app/dashboard/dashboard.component.html

```diff
@@ -8,21 +8,46 @@
 <div class="flex-container">
   <div class="balance">
     <div>Wykres</div>
-    <div>info</div>
+    <div>
+      <p>Godziny Przepracowane: {{ balanceData.hoursWorked }}</p>
+      <p>Minuty Przepracowane:{{ balanceData.minutesWorked }}</p>
+      <p>Godziny Do Przepracowania:{{ balanceData.hoursLeft }}</p>
+      <p>Minuty Do Przepracowania:{{ balanceData.minutesLeft }}</p>
+    </div>
   </div>
 </div>
 
 <div class="flex-container">
   <div class="start-registration">
     <div>Rozpocznij rejetrowanie pracy</div>
-    <button class="clock-in-button">
+    <button (click)="clockIn()" class="clock-in-button">
       <i class="material-icons">access_time</i>
     </button>
+
+    <p *ngIf="responseStatusClockIn !== 200 && responseStatusClockIn !== null">
```

Comment:
Zamiast pisać  !==null czy === null, wystarczy responseStatusClockIn i !responseStatusClockIn.

```suggestion
    <p *ngIf="responseStatusClockIn && responseStatusClockIn !== 200">
```